src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class Reader {
+    public List<String> readCsvFileWithData(Path pathFile) {
+        List<String> result;
+        try {
+            result = Files.readAllLines(pathFile);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + pathFile, e);
+        }
+        return result;
+    }
```

Comment:
The access modifier for the `readCsvFileWithData` method is `public`. If this method is intended to be used only within the package, consider changing it to package-private (no modifier) to adhere to the principle of encapsulation.